lib/apiservers/engine/backends/container.go

```diff
@@ -479,6 +516,11 @@ func (c *Container) containerStart(name string, hostConfig *containertypes.HostC
 			}
 		}()
 
+		// unmap previously mapped ports if they're not powered on
+		err = c.checkActivePortBindings(vc, endpoints)
```

Comment:
This is the wrong set of `endpoints`. The current container does not have any port bindings yet.